### PR TITLE
subhagata haiku

### PR DESCRIPTION
**Pull request by subhagataa**
thank you for reading my pull request :)

### CODE REVIEW

File: _haikus/2022_my first haiku.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: my first haiku
+author: subhagata
+
+It's my first haiku
+I don't even know what to write
+I am still thinking
+
+---
```
Comment (ltagliaferri) [inline _haikus/2022_my first haiku.md:10]:
```suggestion
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
The formatting is not quite right here, I made the suggested edits, can you re-commit?

**Comment by subhagataa** [comment]
How do I re-commit? I clicked those commit  buttons..and resolved the
conversation. Is that it?

File: _haikus/2022_my first haiku.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+layout: haiku
+title: my first haiku
+author: subhagata
+---
+
+It's my first haiku
+I don't even know what to write
```
Comment (ltagliaferri) [inline _haikus/2022_my first haiku.md:8]: Awesome, you re-committed! This line needs one fewer syllable, maybe like this: 

```suggestion
I do not know what to write
```

**Comment by subhagataa** [comment]
done, thank you

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**